docs/source/migrating.rst

```diff
@@ -5,7 +5,7 @@
 Migrating from IPython
 ======================
 
-`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+`The Big Split <http://blog.jupyter.org/2015/04/15/the-big-split/>`__
```

Comment:
@minrk Will do. Oddly, ``make linkcheck`` complains about the SSL. I will change the docs back to https and try to figure out why ``make linkcheck`` errors.